usaspending_api/agencies/tests/test_load_agencies.py

```diff
@@ -2,49 +2,294 @@
 
 from django.core.management import call_command
 from django.core.management.base import CommandError
+from model_mommy import mommy
 from pathlib import Path
-from usaspending_api.agencies.models import CGAC, FREC, SubtierAgency, ToptierAgency
+from usaspending_api.accounts.models import TreasuryAppropriationAccount
+from usaspending_api.agencies.management.commands.load_agencies import Command, Agency as AgencyTuple
+from usaspending_api.agencies.models import CGAC, FREC
+from usaspending_api.awards.models import Award, Subaward, TransactionFPDS, TransactionNormalized
+from usaspending_api.references.models import Agency, SubtierAgency, ToptierAgency
 
 
 AGENCY_FILE = Path(__file__).resolve().parent / "test_load_agencies.csv"
-BOGUS_ABBREVIATION = "THIS IS MY TEST ABBREVIATION"
+BOGUS_ABBREVIATION = "THIS IS A TEST ABBREVIATION"
 
 
-@pytest.mark.django_db
-def test_load_agencies(monkeypatch):
-
-    # Can't run vacuums in a transaction.  Since tests are run in a transaction, we'll NOOP the
-    # function that performs the vacuuming.
+@pytest.fixture
+def disable_vacuuming(monkeypatch):
+    """
+    We cannot run vacuums in a transaction.  Since tests are run in a transaction, we'll NOOP the
+    function that performs the vacuuming.
+    """
     monkeypatch.setattr(
-        "usaspending_api.agencies.management.commands.load_agencies_new.Command._vacuum_tables", lambda a: None
+        "usaspending_api.agencies.management.commands.load_agencies.Command._vacuum_tables", lambda a: None
+    )
+
+
+def _get_record_count():
+    return (
+        ToptierAgency.objects.count()
+        + SubtierAgency.objects.count()
+        + Agency.objects.count()
+        + CGAC.objects.count()
+        + FREC.objects.count()
```

Comment:
Is there a reason to use `objects.count()` here and use `objects.all().count()` in the `test_happy_path` function below? Is there a difference?

Reply:
The difference involves sleep deprivation.  They are the same.